src/mn_setup.c

```diff
@@ -1844,6 +1842,10 @@ static setup_menu_t stat_settings1[] = {
 
     {"Animated Health/Armor Count", S_ONOFF, H_X, M_SPC, {"hud_animated_counts"}},
 
+    MI_GAP,
+
+    {"Weapon Carousel", S_ONOFF, H_X, M_SPC, {"weapon_carousel"}},
```

Comment:
This is temporary. It's tough to pick the right place in the menu for this item, so let's not focus on it for now. I'll take a look at this menu in a separate PR.

Reply:
Okay, but why "Off" by default?

Reply:
It's a neat feature, but I think it should be off by default for the following reasons:
- Visually busy with large, garish icons that overlap or crowd other widgets, messages, and the automap.
- Mostly benefits new players or gamepad users, who I suspect make up a very small portion of Woof's audience.
- No graceful solution for existing mods that change weapons (e.g. detecting if a pwad has custom weapons and using the pickup sprites if it's missing carousel icons).
- The other widgets are disabled by default except automap coords.
- I prefer it off by default. 🙂

Reply:
> * Visually busy with large, garish icons that overlap or crowd other widgets, messages, and the automap.

We can move it lower or make it translucent with SBARDEF settings.

> * No graceful solution for existing mods that change weapons (e.g. detecting if a pwad has custom weapons and using the pickup sprites if it's missing carousel icons).

Players who like alternative weapon sets use GZDoom. Most compatible PWADs use standard weapons or weapons with subtle modifications (Valiant, Eviternity).

> * The other widgets are disabled by default except automap coords.

We should turn off the automap coords as well 😄 

> * I prefer it off by default. 🙂

I really like it! Especially now with animation and without the "stuck" problem.

Reply:
Let's please keep it off by default. It's entirely un-Vanilla-Doomy, and just because the kex port has it enabled doesn't mean we should run behind.

Reply:
> We should turn off the automap coords as well 😄

They were enabled in MBF at least.

Reply:
> Let's please keep it off by default. It's entirely un-Vanilla-Doomy, and just because the kex port has it enabled doesn't mean we should run behind.

But `next/prev` is also un-vanilla-doomy, the carousel will never show up if the user does not press these buttons. Personally, I never used `next/prev` before because Boom logic is complicated and there was no visualization. Now it's usable for me.

Reply:
Okay, that's a point.

Reply:
What's the verdict then? On or off by default?

Reply:
I vote "on". Especially since we have `doom_weapon_cycle` turned off by default, i.e. we've already changed the old default behavior.

Reply:
Sounds okay to me. We can always change it again later based on the response by users.